sycl/source/detail/scheduler/scheduler_helpers.hpp

```diff
@@ -0,0 +1,44 @@
+//==---------- scheduler_helpers.hpp - SYCL standard header file -----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl/queue.hpp>
+#include <detail/scheduler/scheduler.hpp>
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+namespace detail {
+
+void initStream(StreamImplPtr Stream, QueueImplPtr Queue) {
+  auto StreamBuf =
+      Scheduler::getInstance().StreamBuffersPool.find(Stream.get());
+  assert((StreamBuf != Scheduler::getInstance().StreamBuffersPool.end()) &&
+         "Stream is unexpectedly not found in pool.");
+
+  auto &FlushBuf = StreamBuf->second->FlushBuf;
+  // Only size of buffer_impl object has been resized.
+  // Value of Range field of FlushBuf instance is still equal to
+  // MaxStatementSize only.
```

Comment:
It could be just me, but it looks like the comment isn't clear.

Reply:
I tried to rephrase it.